internal/pkg/api/server/buffer/dependencies/store.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/validator"
 )
 
+const MaxReceiverCount = 100
```

Comment:
MaxReceiverCountPerProject? alebo dat komentar

Reply:
Mně to přijde ok, nemáme nikde jinde uložený receivery

Reply:
Mne to evokuje celkovy limit receivers v celom stacku (mohol by existovat aj taky limit).
Ale ok, neries, ak to tebe takto dava zmysel.